docs/source/user-guide/installation.md

```diff
@@ -44,8 +49,21 @@ There are no published JAR files yet.
 
 Official source releases can be downloaded from https://dist.apache.org/repos/dist/release/datafusion/
 
-Building from a source release is mostly the same as building directly from the GitHub repository but requires the 
-use of the command `make release-nogit` instead of `make release`. 
+Download the latest Apache DataFusion Comet tarball file from `datafusion-comet-1.2.3` folder. For example 
```

Comment:
nit: maybe we can use `x.y.z` or `*.*.*` instead of `1.2.3` because the sentence is generic, not specific about an example.